### ISSUE TITLE
UI/UX: easy data range setting for invoicing

### ISSUE THREAD

**Issue by clstaudt** [tuttle-dev/tuttle#174]

**Comment by clstaudt** [comment]
@flamesjames  good first issue?

**Comment by flamesjames** [comment]
@clstaudt yes, I will gladly help with this! Thanks a lot. Assigning myself

**Comment by flamesjames** [comment]
@clstaudt sure thing! I will be able to make some headway on these issues this afternoon. Will shoot over any questions I have after diving in

**Comment by clstaudt** [comment]
Please note: A date picker component is [finally coming to flet](https://github.com/flet-dev/flet/pull/930).

**Comment by berecik** [comment]
autocomplete and date range will be soon, I'm working on integrating flutter into Django and I need that types of fields to have generic flet forms

**Comment by flamesjames** [comment]
@clstaudt should I move forward with the development or would you rather the flet feature be used when it's released?

**Comment by clstaudt** [comment]
@flamesjames  I think having a proper DatePicker in flet like in other UI frameworks will simplify this. But not completely replace a design and implementation of this UI feature.

**Comment by clstaudt** [comment]
@flamesjames I think starting with review/testing tickets makes sense. Perhaps the DatePicker component is available very soon.

**Comment by flamesjames** [comment]
@clstaudt yes I agree with you and think that will help me get more familiar with the app as well before starting dev work on it. I'll hop over to #169 validations

**Comment by flamesjames** [comment]
I see https://github.com/flet-dev/flet/pull/930 has been closed.. I will work on the implementation.

**Comment by clstaudt** [comment]
@flamesjames  Has the DatePicker feature been merged? It looks like it has been moved to a new PR: https://github.com/flet-dev/flet/pull/981

**Comment by flamesjames** [comment]
@clstaudt - ahh you are right.. missed that. Where did you find out about https://github.com/flet-dev/flet/pull/981 just for my future knowledge? I didn't see that on the 930 issue..

**Comment by clstaudt** [comment]
@flamesjames By concidence, watching the open pull requests on flet.

**Comment by clstaudt** [comment]
@berecik   Would you like to give us an Estimated Time of Arrival for the DatePicker feature?

**Comment by flamesjames** [comment]
@clstaudt - just confirming we are still waiting for the date picker flet feature to come to work on this... checked the [PR](https://github.com/flet-dev/flet/pull/981) and it's still open...

**Comment by clstaudt** [comment]
Unfortunately yes. Would you like me to propose other tickets?

**Comment by flamesjames** [comment]
> Unfortunately yes @flamesjames. Would you like me to propose other tickets?

@clstaudt Sure maybe 1 smaller one.. I haven't had much free time for projects recently but can take a lower priority issue if that's fine with you
